File: testjavabenchmark/domain/locationbased/latitude/Statics.java
package domain.locationbased.latitude;

import infrastructure.socialmedia.locationbased.latitude.LatitudeException;
//Functional requirement 3.2
//Test case 137: Class domain.locationbased.latitude.Statics is not allowed to use exception infrastructure.socialmedia.locationbased.foursquare.FourSquareException
//Result: TRUE
public class Statics {
	public Statics(){
		try {
			getStatics();			
			//FR5.8
		} catch (LatitudeException e) {
			e.printStackTrace();
		}
	}

	//FR5.8
	public String getStatics() throws LatitudeException{
		//FR5.8
		throw new LatitudeException("No statics available");
	}
}

File: testprojects/csharp/benchmark/domain/locationbased/latitude/Statics.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using CSharpBenchmark.infrastructure.socialmedia.locationbased.latitude;

namespace CSharpBenchmark.domain.locationbased.latitude
{

    //Functional requirement 3.2
    //Test case 137: Class domain.locationbased.latitude.Statics is not allowed to use exception infrastructure.socialmedia.locationbased.foursquare.FourSquareException
    //Result: TRUE
    public class Statics
    {
        public Statics()
        {
            try
            {
                //FR5.8
            }
            catch (LatitudeException e)
            {
                Console.WriteLine(e.StackTrace);
            }
        }
    }
}